src/manage_nsfs/health.js

```diff
@@ -519,13 +525,34 @@ async function is_bucket_storage_path_exists(fs_context, config_data, storage_pa
         } else if (err.code === 'EACCES' || (err.code === 'EPERM' && err.message === 'Operation not permitted')) {
             dbg.log1('Error:  Storage path should be accessible to account: ', storage_path);
             err_code = health_errors.ACCESS_DENIED.error_code;
+        } else if (err.code === 'BucketSetForbiddenBucketOwnerNotExists') {
+            dbg.log1('Error: Bucket account owner should be valid: ', config_data.owner_account);
+            err_code = health_errors.INVALID_ACCOUNT_OWNER.error_code;
         }
         res_obj = get_invalid_object(config_data.name, undefined, storage_path, err_code);
     }
     return res_obj;
 }
 
-
+/**
+ * get_config_file_data_or_error_object return an object containing config_data or err_obj if error occurred
+ * @param {import('../sdk/config_fs').ConfigFS} config_fs
+ * @param {string} owner_account
+ * @returns {Promise<object>}
+ */
+async function get_account_owner_context(config_fs, owner_account) {
+    if (!owner_account) {
+        const detail_msg = `bucket owner account is empty`;
+        throw_cli_error(ManageCLIError.BucketSetForbiddenBucketOwnerNotExists, detail_msg);
+    }
+    // when account owner is invalid method will throw error
+    const owner_account_data = await get_bucket_owner_account_by_id(config_fs, owner_account);
+    if (owner_account_data) {
```

Comment:
please throw a MISSING_ACCOUNT_OWNER error if owner_account_data is undefined, we don't want to return undefined

Reply:
get_bucket_owner_account_by_id() will throw error if owner account is missing

Reply:
so why you check if (owner_account_data) ? is there a possibility it's undefined? if it undefined, what should we do with it?

Reply:
@romayalon this is lo code and that check is removed, get_bucket_owner_account_by_id() is throwing error when owner_account is invalid, we are already doing empty check.